ta4j-core/src/test/java/org/ta4j/core/util/BarAggregatorTest.java

```diff
@@ -0,0 +1,164 @@
+/*******************************************************************************
+ *   The MIT License (MIT)
+ *
+ *   Copyright (c) 2014-2017 Marc de Verdelhan, 2017-2018 Ta4j Organization 
```

Comment:
Invalid header (should be '2017-2019'). Please remove it, it will be added by maven plugin.